website/docs/r/private_dns_ptr_record.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_private_dns_ptr_record"
+sidebar_current: "docs-azurerm-resource-private-dns-ptr-record"
+description: |-
+  Manages a Private DNS PTR Record.
+---
+
+# azurerm_private_dns_ptr_record
```

Comment:
we need to add a link for this page in the sidebar: https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/azurerm.erb#L1654-L1664